src/components/CreateListingPage/CreateListing.js

```diff
@@ -1,11 +1,11 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import { Input, Textarea } from '../../misc/Input';
-import axios from 'axios';
+import { TextInput, Textarea } from '../../misc/Input';
```

Comment:
Not sure, but maybe you could change 'Textarea' to 'TextArea' (according to TextInput)